src/OrchardCore.Modules/OrchardCore.Media/Views/MediaField-Attached.Edit.cshtml

```diff
@@ -3,54 +3,52 @@
 @using OrchardCore.ContentManagement.Metadata.Models
 
 @{
-var settings = Model.PartFieldDefinition.GetSettings<MediaFieldSettings>();
+    var settings = Model.PartFieldDefinition.GetSettings<MediaFieldSettings>();
     var mediaFieldId = Html.IdFor(m => m);
-    }
+}
 
+<script asp-src="~/OrchardCore.Media/Scripts/media.js" asp-name="media" at="Foot" depends-on="admin, vuejs, sortable, vuedraggable"></script>
+<style asp-src="~/OrchardCore.Media/Styles/media.min.css" debug-src="~/OrchardCore.Media/Styles/media.css"></style>
```

Comment:
returned this to the view it belongs to.
the resource manager handles multiple script inclusions by design, so you don't need to add a wrapper to get this on the page only once.